spec/integration_spec.rb

```diff
@@ -94,6 +94,9 @@ def homepage_data
   it "includes front matter tags" do
     page = page_data('/about/')
     expect(page['tags']).to eq(["Jekyll", "test page", "convenient"])
+    expect(page['meta']['layout']).to eq("page")
+    expect(page['meta']['permalink']).to eq("/about/")
+    expect(page['meta']['skip_index']).to eq(true)
```

Comment:
Discussed IRL: duplicate the data (and include a test demonstrating this) to make life easier for machines.